### ISSUE TITLE
How do I set values in the renderer process, when my store was initialized in the main process?

### ISSUE THREAD

**Issue by sderbidge** [sindresorhus/electron-store#132]
I'd like to set values in the renderer process, would that look like this?

**Main Process**

```
ipcMain.handle('setValue', (event, key, value) => {
  store.set(key, value);
});
```
**Renderer Process**

```
ipcRenderer.invoke(
      "setValue",
      "key", "value"
    );
```

**Comment by sindresorhus** [comment]
What have you tried? Have you seen https://github.com/sindresorhus/electron-store#how-do-i-get-store-values-in-the-renderer-process-when-my-store-was-initialized-in-the-main-process ?

**Comment by sderbidge** [comment]
Yes that's what I based the code in the question off of. I was just wondering what it would look like for setting a value.

**Comment by sindresorhus** [comment]
Yes, that is the right pattern. The FAQ covers it: https://github.com/sindresorhus/electron-store#how-do-i-get-store-values-in-the-renderer-process-when-my-store-was-initialized-in-the-main-process

You can `ipcMain.handle` a setter the same way you handle the getter.
